packages/tools/devtools/devtools-view/src/components/SettingsView.tsx

```diff
@@ -40,18 +34,33 @@ export function SettingsView(props: SettingsProps): React.ReactElement {
 			selectedOptions: string[];
 		},
 	): void {
+		if (!setTheme) {
+			return;
+		}
```

Comment:
This bit feels weird... why would we ever get to this point and have no function to change the theme? Does it have to do with the fact that we initialize the theme to undefined when creating it in `ThemeHelper.ts`? Although then I don't see how would we ever get out of that state... don't we keep hitting this return forever, since all calls to `handleThemeChange` exit here?

Reply:
@alexvy86 Yes, it was because the theme was undefined when created. But we can initialize the context to the theme available. I struggled with initializing the setter, but found that it can be a no-op function since the real state-setting function is provided inside the component.

I've updated the code to reflect this.